avocado/core/test.py

```diff
@@ -282,10 +285,10 @@ def record_and_warn(*args, **kwargs):
 
         self.__outputdir = utils_path.init_dir(self.logdir, 'data')
 
-        # For some reason, sometimes, job.config if None here.
+        # For some reason, sometimes, config if None here.
```

Comment:
Yes, this is strange. @richtja are you be able to reproduce this easily? Can we document this as an issue?

Reply:
Thanks for catching this, I miss it in refactoring. This problem is not happening when we use `config` instead of `job.config`. I will change that in force push.